tiledb/sm/fragment/fragment_info.cc

```diff
@@ -268,19 +264,25 @@ Status FragmentInfo::get_non_empty_domain(
 
 Status FragmentInfo::get_non_empty_domain(
```

Comment:
One more for the new coding standard.

Reply:
Same here as above, but I might as well move the functions of `FragmentInfo` to the new coding standard, as they are just a few.